go/ndt7-client/main.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"flag"
+
+	"github.com/apex/log"
+	"github.com/apex/log/handlers/cli"
+	"github.com/m-lab/ndt7-clients/go/ndt7-client/client"
+)
+
+var hostname = flag.String("hostname", "localhost", "Host to connect to")
+var port = flag.String("port", "443", "Port to connect to")
+var insecure = flag.Bool("insecure", false, "Skip TLS verify")
```

Comment:
Recommend a positive condition instead of a negative one.

```go
var secure = flag.Bool("secure", true, "Verify TLS connections.")
```

"not-insecure" is a pretty awkward way of saying "secure".

Or even, instead of using the generic term "secure", how about:

```go
var tlsVerify = flag.Bool("verify-tls", true, "Verify all TLS connections using PKI. Set to false to allow self-signed, spoofed, and hacked certificates.")
```

Reply:
I disagree with this specific suggestion because:

1. I believe we want the default to be secure, not insecure 

2. and the default in golang is to initialize to false

I really do not want someone integrating this code and not noticing. Hence, I'd like to have a secure by default approach for this variable. (FWIW, this is also was go stdlib does).

Because of the above reasoning, the main code flow is going to be the code flow where we do the secure stuff, and the insecure branch is gonna be the exception, as in:

```go
   // main flow
  if insecure {
    // do insecure stuff; tell your kids not to do this at home
  }
  // rest
```

Reply:
SGTM